### PR TITLE
[UK] Pipe operator, Modules, Mix translation

### PR DESCRIPTION
**Pull request by DmytroLukianov**
Added translations for pipe-operator and modules docs

### CODE REVIEW

**Comment by doomspork** [comment]
Happy New Year @DmytroLukianov! Thank you and welcome. I'm going to give the @elixirschool/ukrainian team a chance to review this before I merge 😁

**Comment by brain-geek** [comment]
Wow, This is quite radical!

File: uk/lessons/basics/modules.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+Що ще більш важливо, структури можна зіставляти з асоціативними масивами:
+
+```elixir
+iex> %{name: "Sean"} = sean
+%Example.User{name: "Sean", roles: [:admin, :owner]}
+```
+
+## Композиція
+
+Тепер, коли ми знаємо як створювати модулі і структури в Elixir, давайте розглянемо як підключати наявну функціональність в них за допомогою композиції. Мова надає безліч різних способів взаємодії з іншими модулями. Давайте розглянемо їх докладніше.
```
Comment (doroshenkoss) [inline uk/lessons/basics/modules.md]: "наявну" -> "існуючу" ?

File: uk/lessons/basics/mix.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    ]
+  end
+
+  defp deps do
+    []
+  end
+end
+```
+
+Перший розділ тут - `project`. У ньому ми визначаємо ім'я додатка (`app`), версію програми (`version`), версію мови Elixir (`elixir`), і всі зовнішні залежності (`deps`).
```
Comment (doroshenkoss) [inline uk/lessons/basics/mix.md]: "додатка" -> "застосунку" ?

File: uk/lessons/basics/modules.md (hunk 1 above)
Comment (DmytroLukianov) [reply uk/lessons/basics/modules.md]: Fixed

File: uk/lessons/basics/mix.md (hunk 2 above)
Comment (DmytroLukianov) [reply uk/lessons/basics/mix.md]: Fixed

File: uk/lessons/basics/modules.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+Що ще більш важливо, структури можна зіставляти з асоціативними масивами:
+
+```elixir
+iex> %{name: "Sean"} = sean
+%Example.User{name: "Sean", roles: [:admin, :owner]}
+```
+
+## Композиція
+
+Тепер, коли ми знаємо як створювати модулі і структури в Elixir, давайте розглянемо як підключати існуючу функціональність в них за допомогою композиції. Мова надає безліч різних способів взаємодії з іншими модулями. Давайте розглянемо їх докладніше.
```
Comment (brain-geek) [inline uk/lessons/basics/modules.md]: `в них за допомогою композиції` shouldn't it be `до них за допомогою композиції`?

**Comment by brain-geek** [comment]
@DmytroLukianov Thanks for submitting this, it's awesome!

File: uk/lessons/basics/modules.md (hunk 3 above)
Comment (DmytroLukianov) [reply uk/lessons/basics/modules.md]: Fixed

**Comment by brain-geek** [comment]
@DmytroLukianov Thanks again for these awesome translations!

**Pull request merged**